CO-CONNECT/load_stage.sql

```diff
@@ -0,0 +1,68 @@
+-- 1. Set latest update
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.SetLatestUpdate(
+	pVocabularyName			=> 'CO-CONNECT',
+    pVocabularyDate			=> CURRENT_DATE,
+	pVocabularyVersion		=> 'CO-CONNECT '||TO_CHAR(CURRENT_DATE,'YYYY-MM-DD'),
+	pVocabularyDevSchema	=> 'dev_co_connect'
+	);
+	END $_$;
+
+DO $_$
+BEGIN
+    PERFORM VOCABULARY_PACK.SetLatestUpdate(
+	pVocabularyName			=> 'CO-CONNECT MIABIS',
+    pVocabularyDate			=> CURRENT_DATE,
+	pVocabularyVersion		=> 'CO-CONNECT MIABIS '||TO_CHAR(CURRENT_DATE,'YYYY-MM-DD'),
+	pVocabularyDevSchema	=> 'dev_co_connect',
+    pAppendVocabulary		=> TRUE
+	);
+	END $_$;
+
+
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.SetLatestUpdate(
+	pVocabularyName			=> 'CO-CONNECT TWINS',
+    pVocabularyDate			=> CURRENT_DATE,
+	pVocabularyVersion		=> 'CO-CONNECT TWINS '||TO_CHAR(CURRENT_DATE,'YYYY-MM-DD'),
+	pVocabularyDevSchema	=> 'dev_co_connect',
+	pAppendVocabulary		=> TRUE
+	);
+	END $_$;
+
+
+-- 2. Truncate all working tables
+TRUNCATE TABLE concept_stage;
+TRUNCATE TABLE concept_relationship_stage;
+TRUNCATE TABLE concept_synonym_stage;
```

Comment:
missing two remaining truncations

Reply:
Even though it is not a drug vocab?

Reply:
yes, because the generic_update does not know whether it is a drug vocab or not, and if it sees the data in the tables (for example, if a drug vocab was running in the schema before), then it will try to apply them too